src/Components/BookState.js

```diff
@@ -8,34 +9,34 @@ const BookState = ({
   const dispatch = useDispatch();
 
   return (
-    <div className="container">
-      <div className="details">
+    <div className={styles.container}>
+      <div className={styles.details}>
         <h4>{category}</h4>
         <h3>{title}</h3>
         <p>{author}</p>
         <ul>
           <li>
-            <button className="line" type="button">Comments</button>
+            <button className={styles.line} type="button">Comments</button>
           </li>
           <li>
-            <button className="line" type="button" onClick={() => dispatch(deleteBook(id))}>
+            <button className={styles.line} type="button" onClick={() => dispatch(deleteBook(id))}>
               Remove
             </button>
           </li>
           <li>
-            <button className="button" type="button">Edit</button>
+            <button className={styles.button} type="button">Edit</button>
```

Comment:
- [ ] According to the project requirement Your final design should match Zeplin's design (this includes all elements + the same styling). You can use the  CSS rules detailed in Zeplin to style the website. Please compare the design below and make the necessary changes.
- [ ] Make sure to use the same color, and border and also follow the spacing so It matches the design.
### Your design
![his](https://github.com/tchasinga/bookstoring/assets/81727988/6d237df8-8217-4ff3-89a0-adbfb09bf598)

### Original design
![original](https://github.com/tchasinga/bookstoring/assets/81727988/5b785721-cde7-4bf0-af44-8db910ad15ed)